docs/network-flow-visibility.md

```diff
@@ -354,6 +354,14 @@ the Flow Aggregator adds the following fields to the flow records.
 | reverseOctetTotalCountFromDestinationNode | 56506         | 133      | unsigned64  |
 | reversePacketDeltaCountFromDestinationNode| 56506         | 134      | unsigned64  |
 | reverseOctetDeltaCountFromDestinationNode | 56506         | 135      | unsigned64  |
+| throughput                                | 56506         | 145      | unsigned64  |
```

Comment:
not in this PR, can you add the pod label related fields in this table?

Reply:
sure